src/openapi/streaming/connection/transport/signalr-core-transport.js

```diff
@@ -191,27 +199,15 @@ SignalrCoreTransport.prototype.start = function(options, onStartCallback) {
             this.stateChangedCallback(constants.CONNECTION_STATE_CONNECTED);
         })
         .catch((error) => {
-            if (error.statusCode) {
-                log.error(
-                    LOG_AREA,
-                    'Error occurred while connecting to streaming service',
-                    {
-                        error,
-                    },
-                );
-
-                this.transportFailCallback();
-            } else {
-                log.debug(
-                    LOG_AREA,
-                    'Possible network error occurred while connecting to streaming service',
-                    {
-                        error,
-                    },
-                );
-
-                this.handleConnectionClosure();
-            }
+            log.error(
+                LOG_AREA,
+                'Error occurred while connecting to streaming service',
+                {
+                    error,
+                },
+            );
+
+            this.transportFailCallback();

```

Comment:
so.. before it retried and now it fails so that it goes back to the on premise streaming?

I think it could be like this in the beginning, but eventually we are going to probably want it to retry some times and then fallback.

Reply:
Yes.
Moreover I am looking into using fetch util as httpClient for http requests made by signalr lib.

Reply:
what http requests are used by signalr app? what does this have to do with the above?

Reply:
negotiation request 
and signalr use it for long polling for fallback scenario.
our fetch util has isNetworkError as clear api
but I suppose we will be dealing with same problem ?

Reply:
That doesn't change things
So I will keep it as it is